src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -28,17 +28,54 @@ public static void printLongestWordTop100() throws IOException {
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
         // TODO 이 부분에 구현한다.
+        List<String> longestWords = words.stream()
+                .filter(word -> word.length() > 12)
+                .sorted(sortByLength())
+                .distinct()
+                .collect(Collectors.toList());
+
+        List<String> topWords = getTopWords(longestWords);
+        topWords.forEach(System.out::println);
```

Comment:
다음과 같이 해볼 수도 있을 것 같아요! 😄 
```suggestion
        words.stream()
                .filter(word -> word.length() > 12)
                .sorted(Comparator.comparingInt(String::length).reversed())
                .distinct()
                .limit(100)
                .forEach(word -> System.out.println(word.toLowerCase()));
```